src/modules/keyboardmanager/KeyboardManagerEditor/Resources.resx

```diff
@@ -177,14 +177,14 @@
     <value>Some of the shortcuts could not be remapped. Continue anyway?</value>
   </data>
   <data name="EditKeyboard_Info" xml:space="preserve">
-    <value>Select the key you want to change ("Select") and then the key or shortcut you want it to send ("To send").</value>
+    <value>Select the key you want to change ("Select") and then the key, shortcut or text you want it to send ("To send").</value>
   </data>
   <data name="EditKeyboard_InfoExample" xml:space="preserve">
     <value>Example of a remapping: Select A and send "Ctrl+C", "A" would be your "Select" and "Ctrl+C" would be your "To send" command.</value>
     <comment>Key on a keyboard</comment>
   </data>
   <data name="EditShortcuts_Info" xml:space="preserve">
-    <value>Select the shortcut you want to change ("Select") and then the key or shortcut you want it to invoke ("To send").</value>
+    <value>Select the shortcut you want to change ("Select") and then the key, shortcut or text you want it to invoke ("To send").</value>
```

Comment:
```suggestion
    <value>Select the shortcut you want to change ("Select") and then configure the key, shortcut or text you want it to send ("To send").</value>
```

You can't sekect the "target" text. 😅

Why her `to invoke` and for above `to send`. `to send` sounds better because you can't invoke text.